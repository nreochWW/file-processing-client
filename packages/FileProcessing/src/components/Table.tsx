import * as React from "react";
import { useQuery } from "@apollo/react-hooks";
import gql from "graphql-tag";
import Data from "./Data";

const GET_STATUS = gql`
  query {
    status {
      message
    }
  }
`;

export default function Table() {
  const { loading, data } = useQuery(GET_STATUS);
  return (
    <div>
      {loading ? <p>Loading</p> : <p>{JSON.stringify(data)}</p>}
      <h1>File Processing</h1>
      <Data />
    </div>
  );
}
