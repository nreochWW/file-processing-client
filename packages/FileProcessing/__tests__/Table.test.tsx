import * as React from "react";
import { shallow } from "enzyme";
import Table from "../src/components/Table";

describe("Table package tests", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Table />);
    expect(wrapper).toMatchSnapshot();
  });
});
