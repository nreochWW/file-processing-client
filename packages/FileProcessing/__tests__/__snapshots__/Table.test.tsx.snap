// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table package tests matches snapshot 1`] = `
<div>
  <h1>
    File Processing
  </h1>
</div>
`;
